### DISCUSSION TITLE
Is there a way to resize the width of the spotlight

### DISCUSSION THREAD

**Discussion by vaishawil** [gilbarbara/react-joyride#636] [Q&A] [answered]
Hi, i just found out that the spotlight would cover the targeted component based on it's width. therefore i would to know if there is a way to resize the width of the spotlight? 

The illustration probably looks like this, the black box are the content inside the spotlight, because there is an excess of whitespace, i was thinking to reduce the width of the spotlight to fit the content only
![SCREENSHOT](https://user-images.githubusercontent.com/77441574/105025376-d375ef00-5a7f-11eb-9ca5-83804ccad5cb.png)

**Comment by gilbarbara** [comment, marked as answer]
Nope. You can change the padding but not the width.
Think responsive...

[s]

**Reply by vaishawil** [reply]
okay, thanks for the information.
